.gitignore

```diff
@@ -109,7 +109,7 @@ share
 .sqlite
 .sqlite3
 venv
-static
+/static
```

Comment:
What is the idea/reason behind this initial slash ?

Reply:
The reason I added the '/' before 'static' in the .gitignore file is to ensure that only the static folder in the main project directory is excluded, while preserving the static folders inside individual apps. This approach allows us to manage the static files, ensuring that each app can maintain its own static resources without interference.

In the workflow, the static folder in the main project directory is typically managed using the 'python3 manage.py createstatic' command. This command is necessary after every change in static files to update the main project's static folder. By excluding the static folder in the main project directory from version control, we prevent unnecessary duplication or conflicts with other static folders.